### ISSUE TITLE
Accessibility hooks for charts; Screen-reader usability. 

### ISSUE THREAD

**Issue by ish-joshi** [PhilJay/MPAndroidChart#4890]
**Is your feature request related to a problem? Please describe.**

A very different set of users cannot experience MPAndroidChart. The library lacks custom accessibility hooks that enables screen-readers to read data. I have built some proof-of-concepts that reads out basic statistics about the data and passes it to the screenreader. I am willing to have a discussion about what the best way to implement is and take it from there. 

**Describe the solution you'd like** 

Custom accessibility hooks for custom views. I have a POC that gets access to raw data and on focus of the chart it will send the textual description (generated on the fly) to the screen-reader service. 

**Describe alternatives you've considered**
Not many, it'd require an assistant to describe the data or rely on textual description as part of the development. 

**Additional context**

ADD A REWARD using [Speed](speedoss.com) to SOLVE this issue QUICKLY and SUPPORT this project.
